test/minigzip.c

```diff
@@ -250,20 +250,20 @@ void file_uncompress(char *file, int keep) {
         unlink(infile);
 }
 
-
-/* ===========================================================================
- * Usage:  minigzip [-c] [-d] [-f] [-h] [-r] [-1 to -9] [files...]
- *   -c : write to standard output
- *   -d : decompress
- *   -k : Keep input files
- *   -f : compress with Z_FILTERED
- *   -h : compress with Z_HUFFMAN_ONLY
- *   -R : compress with Z_RLE
- *   -F : compress with Z_FIXED
- *   -T : stored raw
- *   -A : auto detect type
- *   -0 to -9 : compression level
- */
+void show_help(void)
+{
+    printf("Usage: minigzip [-c] [-d] [-k] [-f|-h|-R|-F|-T] [-A] [-1 to -9] [files...]\n\n" \
```

Comment:
Should be "-0 to -9" to be consistent with the text below.

Reply:
Fixed.